independent-publisher-connectors/xkcd/readme.md

```diff
@@ -0,0 +1,19 @@
+# xkcd
+xkcd, sometimes styled XKCD, is a webcomic created in 2005 by American author Randall Munroe. The comic's tagline describes it as "a webcomic of romance, sarcasm, math, and language".
+
+## Publisher: Troy Taylor, Hitachi Solutions
+
+## Prerequisites
+There are no prerequisites for this connector.
+
+## Obtaining Credentials
+There are no credentials needed for this connector.
+
+## Supported Operations
+### Get current comic
+Get the current comic. xkcd is updated every Monday, Wednesday, and Friday.
+### Get a specific comic
+Returns a specific comic by comic number.
+
+## Known Issues and Limitations
+There are no known issues at this time.
```

Comment:
There is no Deployment Instruction section. Please refer to this [link ](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors)as a template for Readme.md.

Reply:
Optional